components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -158,8 +157,8 @@ const InfoListItemRender: React.ForwardRefRenderFunction<unknown, InfoListItemPr
     } = props;
 
     const combine = useCallback(
-        (className: keyof InfoListItemClasses): string => cx(defaultClasses[className], classes[className]),
-        [defaultClasses, classes]
+        (className: keyof InfoListItemClasses): string => cx(generatedClasses[className]),
+        [generatedClasses]
     );
```

Comment:
This function does nothing now (just does a bunch of lookups to return the exact same thing you passed into it) — I would probably recommend just removing it, especially if it's only in 3 components.